lib/errinj.c

```diff
@@ -0,0 +1,104 @@
+/**
+ *		Tempesta FW
+ *
+ * Error injection library.
+ *
+ * Application protocol handler layers must implement zero data copy logic
+ * on top on native Linux socket buffers. The helpers provide common and
+ * convenient wrappers for skb processing.
+ *
+ * Copyright (C) 2015-2023 Tempesta Technologies, Inc.
```

Comment:
Usually for new files we use only current year, so there should be just `2023`. If you see `2015-2023`, then it's quite an old file ;)

Reply:
Fixed